### ISSUE TITLE
False positive: emptyrpm inspection reports package as empty when it contains files

### ISSUE THREAD

**Issue by fmarco76** [rpminspect/rpminspect#1494]
**Describe the bug**

The `emptyrpm` inspection incorrectly reports a valid RPM package as empty when the package clearly contains files and payload data. The package I have problem with is [dogtag-pki-server-11.9.0~alpha1%5E20250807171113.6502a25d-1.fc42.noarch.rpm](https://download.copr.fedorainfracloud.org/results/@pki/master/fedora-42-x86_64/09387191-pki/dogtag-pki-server-11.9.0~alpha1%5E20250807171113.6502a25d-1.fc42.noarch.rpm)

**To Reproduce**
The command to reproduce the error is:
```
# usr/bin/rpminspect -c /usr/share/rpminspect/fedora.yaml -T emptyrpm dogtag-pki-server-11.9.0~alpha1\^20250807171113.6502a25d-1.fc42.noarch.rpm 
rpminspect: *** archive_read_extract: Hard-link target '/var/tmp/rpminspect/local.Iliyn9/root/after/noarch/(null)' does not exist.
diagnostics:
------------
1) Version information for libraries and programs used by rpminspect as well as storage requirements. 
This result is for informational and diagnostic purposes only. 

Result: DIAGNOSTICS
Details:
rpminspect version 2.0
zlib version 1.3.1.zlib-ng
libmagic version 546
libclamav version 1.4.3
librpm version 4.20.1
libxml version 21210
json-c version 0.18
libcurl version 8.11.1 (OpenSSL/3.2.4 zlib/1.3.1.zlib-ng brotli/1.1.0 libidn2/2.3.8 libpsl/0.21.5 libssh/0.11.2/openssl/zlib nghttp2/1.64.0 OpenLDAP/2.6.9)
libarchive version 3.7.7 (zlib/1.3.1.zlib-ng liblzma/5.6.3 bz2lib/1.0.8 liblz4/1.10.0 libzstd/1.5.6)
libyaml version 0.2.5
OpenSSL version 3.2.4 11 Feb 2025
xmlrpc-c version 1.60.4
msgunfmt (GNU gettext-tools) 0.23.1
annocheck version 12.94.
abidiff version 2.8.0
kmidiff version 2.8.0
udevadm version 257

Space required to download artifacts: 0 bytes (0 B)
Space required to unpack artifacts: 10103636 bytes (10 MiB)



2) Command line arguments used to invoke rpminspect. 

Result: DIAGNOSTICS
Details:
usr/bin/rpminspect -c /usr/share/rpminspect/fedora.yaml -T emptyrpm dogtag-pki-server-11.9.0~alpha1^20250807171113.6502a25d-1.fc42.noarch.rpm


3) Build 

Result: DIAGNOSTICS
Details:
dogtag-pki-server-11.9.0~alpha1^20250807171113.6502a25d-1.fc42.noarch.rpm



abidiff:
--------
Result: SKIP


addedfiles:
-----------
Result: SKIP


annocheck:
----------
Result: SKIP


arch:
-----
Result: SKIP


badfuncs:
---------
Result: SKIP


capabilities:
-------------
Result: SKIP


changedfiles:
-------------
Result: SKIP


changelog:
----------
Result: SKIP


config:
-------
Result: SKIP


debuginfo:
----------
Result: SKIP


desktop:
--------
Result: SKIP


disttag:
--------
Result: SKIP


doc:
----
Result: SKIP


dsodeps:
--------
Result: SKIP


elf:
----
Result: SKIP


emptyrpm:
---------
1) New package dogtag-pki-server-11.9.0~alpha1^20250807171113.6502a25d-1.fc42.noarch.rpm is empty (no
payloads) 

Result: VERIFY
Waiver Authorization: Anyone

Suggested Remedy: 


files:
------
Result: SKIP


filesize:
---------
Result: SKIP


javabytecode:
-------------
Result: SKIP


kmidiff:
--------
Result: SKIP


kmod:
-----
Result: SKIP


license:
--------
Result: SKIP


lostpayload:
------------
Result: SKIP


lto:
----
Result: SKIP


manpage:
--------
Result: SKIP


metadata:
---------
Result: SKIP


modularity:
-----------
Result: SKIP


movedfiles:
-----------
Result: SKIP


ownership:
----------
Result: SKIP


patches:
--------
Result: SKIP


pathmigration:
--------------
Result: SKIP


permissions:
------------
Result: SKIP


politics:
---------
Result: SKIP


removedfiles:
-------------
Result: SKIP


rpmdeps:
--------
Result: SKIP


runpath:
--------
Result: SKIP


shellsyntax:
------------
Result: SKIP


specname:
---------
Result: SKIP


subpackages:
------------
Result: SKIP


symlinks:
---------
Result: SKIP


types:
------
Result: SKIP


udevrules:
----------
Result: SKIP


unicode:
--------
Result: SKIP


upstream:
---------
Result: SKIP


virus:
------
Result: SKIP


xml:
----
Result: SKIP
```

The package is not empty:

```bash
$ rpm -qlp dogtag-pki-server-*.rpm | wc -l
917

$ rpm -qlp dogtag-pki-server-*.rpm | head -5
/etc/pki/healthcheck.conf
/etc/sysconfig/pki
/etc/sysconfig/pki/tomcat
/etc/systemd/system/pki-tomcatd-nuxwdog.target.wants
/etc/systemd/system/pki-tomcatd.target.wants

$ rpm -qip dogtag-pki-server-*.rpm
Name        : dogtag-pki-server
Version     : 11.9.0~alpha1
Release     : 1.fc42
Architecture: noarch
[... shows normal package metadata ...]
```

**Expected behavior**
The `rpminspect` should not report the package as empty.

**Environment:**
 - Test are executed in a clean Fedora 42 on x86_64 architecture. Packages version  as reported in the command above.
